src/vni-manager.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * File:  vni-manager.js
+ */
+var _ = require('underscore');
+
+var inputStates = require('./input-states');
+
+var IIP_VNID = '';
+
+/**
+ * Manage the node's vni's by vnid.  If no vnid is provided the IIP VNID will be used.
+ * If the vni for the specified vnid does not exist, a new one will be created.
+ *
+ * @this the component node instance unless otherwise specified on an API
+ * @param facade a component facade
+ * @return the component facade, extended to include the vni interface
+ * 
+ * Usage examples: 
+ *     var vni = this.nodeInstance.vni(); // retrieve an IIP vni 
+ *     var vni = this.nodeInstance.vni(vnid); // retrieve the vni associated with the vnid
+ *     var context = component.deleteVnis();
```

Comment:
This should be node.deleteVnis() -- not component.deleteVnis() -- because the VNIs are associated with the node (instances), not the component (factory).